docs/release-notes/RELEASE-NOTES-0.9.0.md

```diff
@@ -1,50 +1,63 @@
 # About this release
 
-[This release](https://github.com/aeternity/epoch/releases/tag/v0.9.0) is focused on TODOFILLMEIN.
+[This release][this-release] is focused on TODOFILLMEIN.
 It:
 * Does this TODOFILLMEIN. This impacts consensus;
 * Does that TODOFILLMEIN. This impacts the persisted DB;
 * Does that TODOFILLMEIN.
 
-This release introduces backward incompatible changes in the chain format.
-After upgrading your node, you will not have your previous balance (even if you keep your key pair).
+[this-release]: https://github.com/aeternity/epoch/releases/tag/v0.9.0
+
+This release introduces backward incompatible changes in the chain format:
+* After upgrading your node, you will not have your previous balance (even if you keep your key pair);
+* Please ensure that you do not reuse a persisted blockchain produced by the previous releases "v0.8.x".
 
 Please join the testnet by following the instructions below, and let us know if you have any problems by [opening a ticket](https://github.com/aeternity/epoch/issues).
 Troubleshooting of common issues is documented [in the wiki](https://github.com/aeternity/epoch/wiki/Troubleshooting).
 
 The instructions below describe:
 * [How to retrieve the released software for running a node](#retrieve-the-software-for-running-a-node);
+* [How to install a node](#install-node);
 * [How to join the testnet](#join-the-testnet).
 
 ## Retrieve the software for running a node
 
-Download the [release binary](https://github.com/aeternity/epoch/releases/tag/v0.9.0) corresponding to your platform, e.g. `epoch-0.9.0-osx-10.12.6.tar.gz`; you would normally find the downloaded package in `~/Downloads` on macOS.
+You can run a node by using either:
+* The published [release binary][this-release] corresponding to your platform; or
+* The published [Docker image `aetrnty/epoch`][docker].
 
-The binaries are tested on the following platforms:
-* Ubuntu 16.04.3 LTS (x86-64);
-* macOS Sierra (x86-64);
-* macOS High Sierra (x86-64).
+[dockerTODO]: https://github.com/aeternity/epoch/blob/master/v0.9.0/docker.md
```

Comment:
todo?

Reply:
> Also propose make most links point to current branch during development.

I propose that links link to master during development, and we change them to tag before release. These TODOs are meant to be placeholders of which links shall be amended. If it looks too complex I am happy to revert back to hardcoding tag number (e.g. 0.9.0) even during development.

Reply:
BTW your comment made me see a typo, that I fixed in 98cbabc